### PR TITLE
[Camera] Add new camera setting APIs

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add new camera setting APIs like the followings.
- Saturation
- Sharpness
- Gain
- WhiteBalanceTemperature

### API Changes ###

 - TCSACR-538

### CODE REVIEW

File: src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs [hunk 1]
```diff
@@ -375,6 +383,214 @@ public Range ExposureRange
...
+            }
+            set
+            {
+                Native.SetSaturation(_camera.GetHandle(), value).
+                    ThrowIfFailed("Failed to set camera saturation value.");
+            }
+        }
+
+        /// <summary>
+        /// Gets the available saturation value.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs:409]: that follows the pattern of already existing properties, but "Gets the available saturation value range." or "Gets the available saturation range." or "Gets the range of allowed saturation value."  would be better.

File: src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs [hunk 2]
```diff
@@ -375,6 +383,214 @@ public Range ExposureRange
...
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        /// <remarks>
+        /// If the minimum value is greater than the maximum value, it means this feature is not supported.
+        /// </remarks>
+        /// <exception cref="NotSupportedException">Saturation is not supported.</exception>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs:415]: lines 413 and 415 suggest 2 different outcomes when saturation is not supported. Which is it then - an empty range or exception?

**Review by wiertel — APPROVED** [review]

File: src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs:409]: I'll fix it.

File: src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs:415]: As described in remarks, If min value is greater than the max value, it means XXRange is not supported and in this case, NotSupportedException will be thrown. 
Those values are tested by `GetRange()` method in 64 line.

Comment (wiertel) [reply src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs:415]: The thing is that it's not clear to me what will happen when the feature is not supported - It cannot be both an exception and some values.  Will app developer get an exception or an empty Range object?

**Pull request merged**